tests/func/test_download.py

```diff
@@ -0,0 +1,83 @@
+import os
+from unittest.mock import ANY
+
+import pytest
+
+from dvc.fs import download
+
+lfs_prefetch_params = [
+    pytest.param("abc", "abc", id="plain"),
+    pytest.param(
+        "*",
+        "[*]",
+        marks=pytest.mark.skipif(
+            os.name == "nt",
+            reason="forbidden character `*` on Windows filesystem",
+        ),
+        id="escape-*",
+    ),
+    pytest.param(
+        "**",
+        "[*][*]",
+        marks=pytest.mark.skipif(
+            os.name == "nt",
+            reason="forbidden character `*` on Windows filesystem",
+        ),
+        id="escape-**",
+    ),
+    pytest.param(
+        "?",
+        "[?]",
+        marks=pytest.mark.skipif(
+            os.name == "nt",
+            reason="forbidden character `?` on Windows filesystem",
+        ),
+        id="escape-?",
+    ),
+    pytest.param(
+        "[abc]",
+        "[[]abc]",
+        id="escape-[seq]",
+    ),
+    pytest.param(
+        "[!abc]",
+        "[[]!abc]",
+        id="escape-[!seq]",
+    ),
```

Comment:
Each of those params calls should be formatted on one line without line wrap – copy & paste error + Ruff formatter can't unwrap lines. Let me fix this first thing tomorrow morning.

Reply:
Done.